packages/php/email-editor/src/Engine/Renderer/class-html2text.php

```diff
@@ -0,0 +1,622 @@
+<?php
```

Comment:
Original file for comparison: https://github.com/soundasleep/html2text/blob/master/src/Html2Text.php